### PR TITLE
Prepare for release v6.3.3

### PR DESCRIPTION
**Pull request by shervElmi**
### Changes proposed in this Pull Request:
With this Pull Request, we will release version 6.3.3.

### Changelog:
- Fix: Listing titles were not always showing.
- Fix: Resolve some CSS issues on the single page.

### CODE REVIEW

**Review by stephywells — CHANGES_REQUESTED** [review]

File: README.TXT [hunk 1]
```diff
@@ -158,6 +158,10 @@ Yes it is. However, you cannot "network activate" the plugin (as this will share
 This can be done under Plugins -> Add New as the Administrator user. Do not "network activate" as the "super admin".
 
 == Changelog ==
+= 6.3.3 =
+* Fix: Listing titles were not always showing.
+* Fix: Resolve some CSS issues on the single page.
```
[the remark this answers is not in the document]
Comment (shervElmi) [reply README.TXT:163]: > Can you please merge my latest commits too? For releases, we need to either include everything scheduled for the milestone, or move them to a different one.

Sure! Ok, got it.

[the remark this answers is not in the document]
Comment (stephywells) [reply README.TXT:163]: Thank you! Let's add it to the changelog too.

[the remark this answers is not in the document]
Comment (shervElmi) [reply README.TXT:163]: You're welcome! Is the change log "Fix: The menu height when the admin menu is short."?

[the remark this answers is not in the document]
Comment (shervElmi) [reply README.TXT:163]: @stephywells I've updated the changelog.

[the remark this answers is not in the document]
Comment (stephywells) [reply README.TXT:163]: That seems a little hard to understand. Maybe:
Fix: The BD sidebar didn't fill the page in cases where the WP admin nav was short

**Comment by shervElmi** [comment]
@stephywells Would you please approve? thanks.

**Pull request merged**